solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/LoadingView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  LoadingView.swift
+//  DeliveryApp
+//
+//  Created by Hyago Henrique on 21/02/22.
+//
+
+import UIKit
+
+final class LoadingView: UIView {
+
+    // MARK: - View Components
+
+    private let loadingContainer: UIView = {
```

Comment:
Para garantir uma maior performance/otimização do seu código, considere utilizar `lazy var` para suas propriedades de componentes visuais.

Este artigo explica bem quando utilizar `lazy var`: https://www.avanderlee.com/swift/lazy-var-property/#:~:text=Lazy%20variables%20allow%20you%20to,the%20current%20state%20of%20values.

Além das `lazy vars` otimizarem a inicialização ocorrendo somente quando a propriedade for utilizada pela primeira vez, ela ainda garante que o valor inicial não tenha que ser recalculado toda vez que você chama a sua propriedade, garantindo assim uma maior performance e experiência.

Reply:
show, vou dar uma lida nesse artigo.